### DISCUSSION TITLE
How to add Chart js to Quasar project and use it ?

### DISCUSSION THREAD

**Discussion by amirsafaricg** [quasarframework/quasar#10707] [Ideas / Proposals]
How can I use chartjs in quasar project without any error's and issue's ? 
I installed it like a normal vue project but it didn't work 
does anyone know how to set it up?

**Comment by pikil** [comment]
```
<template>
   <canvas ref="chartPie" class="full-width full-height" />
</template>
<script>
import {
  Chart,
  PieController
} from 'chart.js'

Chart.register(
  PieController
)


...

  setup () {
      new Chart(chartPie.value.getContext('2d'), {
        type: 'pie',
        data: {
          labels: ['Open', 'Completed', 'Doing'],
          datasets: [
            {
              data: [50, 30, 20]
            }
          ]
        }
      })
   }
</script>
```

That should be enough for jumpstart,
do not forget about ```const chartPie = ref(null)```

**Comment by ewertonrfd** [comment]
It's not the chartJS 😄 

But it worked well for me
[vue3-apexcharts](https://github.com/apexcharts/vue3-apexcharts)

To use very simple, especially in vue 3 😃  with Quasar 2

**Reply by ewertonrfd** [reply]
https://github.com/quasarframework/quasar/discussions/10614
